cli/TODO_HTTP_API.md

```
@@ -0,0 +1,43 @@
+# CLI HTTP API migration TODOs
+
+## Missing endpoints (block full supabase-js removal)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)